crates/cibse_guide_e/src/chapter_7/equation_7_9/openfire_runner.rs

```diff
@@ -0,0 +1,215 @@
+use framework::method::runner::MethodRunner;
+use framework::method::tag::Tag;
+use framework::method::test::IntegrationTests;
+use framework::method::validation::ParameterError;
+use framework::method::{Method, step::Step};
+
+use crate::CIBSEGuideE;
+
+use std::sync::{Arc, RwLock};
+use std::vec;
+
+struct Symbols {
+    n_in: &'static str,
+    t: &'static str,
+    w_e: &'static str,
+    p: &'static str,
+    a: &'static str,
+    s: &'static str,
+}
+
+const SYMBOLS: Symbols = Symbols {
+    n_in: "N_{in_{max}}",
+    w_e: "W_e",
+    t: "t",
+    p: "p",
+    a: "A",
+    s: "S",
+};
+
+#[derive(Default)]
+pub struct Chapter7Equation9Runner;
+
+impl MethodRunner for Chapter7Equation9Runner {
+    fn name(&self) -> String {
+        "Acceptance capacity of the stair".to_string()
+    }
+    fn reference(&self) -> &dyn framework::method::runner::Reference {
+        &CIBSEGuideE::ChapterSeven(crate::chapter_7::Chapter7Method::Equation7_9)
+    }
+    fn tags(&self) -> Vec<Tag> {
+        vec![Tag::Evacuation]
+    }
+    fn description(&self) -> Option<String> {
+        Some(
+            "Maximum number of people able to enter the stair within a specified period"
+                .to_string(),
+        )
+    }
+    fn quick_calc(&self, params: &Parameters) -> Option<Vec<ArcParameter>> {
+        let n_in = params.get(SYMBOLS.n_in);
+
+        Some(vec![n_in])
+    }
+
+    fn form(&self, params: &Parameters) -> framework::method::form::Form {
+        let n_in = params.get(SYMBOLS.n_in);
+        let w_e = params.get(SYMBOLS.w_e);
+        let t = params.get(SYMBOLS.t);
+        let p = params.get(SYMBOLS.p);
+        let a = params.get(SYMBOLS.a);
+        let s = params.get(SYMBOLS.s);
+
+        let mut step_1 = FormStep::new("Input | Eq. 7.9", "Acceptance capacity of a stairway");
+        step_1.add_field(w_e.to_field());
+        step_1.add_field(p.to_field());
+        step_1.add_field(t.to_field());
+        step_1.add_field(a.to_field());
+        step_1.add_field(s.to_field());
+
+        step_1.add_intro();
+        step_1.add_equation(CalculationComponent::Equation(super::equation(
+            n_in.symbol(),
+            w_e.symbol(),
+            t.symbol(),
+            p.symbol(),
+            a.symbol(),
+            s.symbol(),
+        )));
+
+        Form::new(vec![step_1])
+    }
+    fn parameters(&self) -> Parameters {
+        let mut params = Parameters::new();
+
+        let n_in = ParamBuilder::integer(SYMBOLS.n_in)
+            .name("Maximum number of people able to enter the stair within a specified period")
+            .units("persons")
+            .build();
+
+        let w_e = ParamBuilder::float(&SYMBOLS.w_e)
+            .name("Effective width of the stair")
+            .units("m")
+            .min_exclusive(0.0)
+            .required()
+            .build();
+
+        let t = ParamBuilder::float(&SYMBOLS.t)
+            .name("Available exiting time")
```

Comment:
```suggestion
            .name("Time available for escape")
```

Another frustrating difference between this and equation 7.8. The wording used for this variable is different in each. Lets keep it consistant I think even though Cibse does not

Reply:
Agreed. I hesitated when I filled it out and ended up going for the CIBSE E description, but you are right, if they are the same they should say the same.